langlib/lang.xml/src/main/ballerina/xml.bal

```diff
@@ -131,6 +178,11 @@ public isolated function setName(Element elem, string xName) = @java:Method {
 # This includes namespace attributes.
 # The keys in the map are the expanded names of the attributes.
 #
+# ```ballerina
+# xml:Element a = xml `<person gender="male" isEmployed="yes"><name>John</name></person>`;
+# a.getAttributes() ⇒ {"gender":"male","isEmployed":"yes"}
```

Comment:
```suggestion
# xml:Element a = xml `<person id="1012" employed="yes"><name>John</name></person>`;
# a.getAttributes() ⇒ {"id":"1012","employed":"yes"}
```

Reply:
Updated in https://github.com/ballerina-platform/ballerina-lang/pull/38981/commits/9dc46b7c36c28b92b9ce360ed71621b3a6eff8c0